pkg/activator/handler/requesttimemetric_handler.go

```diff
@@ -0,0 +1,78 @@
+/*
+Copyright 2019 The Knative Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package handler
+
+import (
+	"net/http"
+	"time"
+
+	"go.uber.org/zap"
+
+	"knative.dev/pkg/logging/logkey"
+	"knative.dev/serving/pkg/activator"
+	"knative.dev/serving/pkg/apis/serving"
+	servinglisters "knative.dev/serving/pkg/client/listers/serving/v1alpha1"
+	pkghttp "knative.dev/serving/pkg/http"
+)
+
+// NewRequestTimeMetricHandler creates a handler that sends metrics to autoscaler
```

Comment:
Does it sent them to autoscaler, or rather to prometheus? (I am pretty sure the latter)